packages/edit-post/src/components/sidebar/plugin-document-setting-panel/index.js

```diff
@@ -33,7 +38,7 @@ const PluginDocumentSettingFill = ( { isEnabled, panelName, opened, onToggle, cl
 						opened={ opened }
 						onToggle={ onToggle }
 					>
-						{ children }
+						{ ( isFunction( children ) ) ? ( children( { currentPostType } ) ) : ( children ) }
```

Comment:
The extra parentheses are unnecessary. I personally don't think they help add clarity, either.

```suggestion
						{ isFunction( children ) ? children( { currentPostType } ) : children }
```

Reply:
Updated